tensorflow-framework/src/main/java/org/tensorflow/framework/activations/ELU.java

```diff
@@ -72,16 +76,96 @@ public ELU(double alpha) {
     this.alpha = alpha;
   }
 
-  /** {@inheritDoc} */
-  @Override
-  public Operand<T> call(Ops tf, Operand<T> input) {
+  /**
+   * Creates a new ELU from a configuration Map
+   *
+   * @param config the configuration map, if the map contains an entry for {@code alpha} that value
+   *     is used, otherwise {@link #ALPHA_DEFAULT} is used.
+   */
+  public ELU(Map<String, Object> config) {
+    this((Double) config.getOrDefault("alpha", ALPHA_DEFAULT));
+  }
+
+  /**
+   * Computes the Exponential linear unit.
+   *
+   * <p>The exponential linear unit (ELU) with <code>alpha &gt; 0</code> is:
+   *
+   * <p><code>x</code> if <code>x &gt; 0</code> and <code>alpha * (exp(x) -
+   *  * 1)</code> if <code>x &lt; 0</code>.
+   *
+   * <p>The ELU hyperparameter <code>alpha</code> controls the value to which an ELU saturates for
+   * negative net inputs. ELUs diminish the vanishing gradient effect.
+   *
+   * <p>ELUs have negative values which pushes the mean of the activations closer to zero. Mean
+   * activations that are closer to zero enable faster learning as they bring the gradient closer to
+   * * the natural gradient. ELUs saturate to a negative value when the argument gets smaller.
```

Comment:
I think the `*` got doubled here, and in one of the code sections above).

Reply:
I just did a clean up of the java doc for activations